src/stdio_mgr/stdio_mgr.py

```diff
@@ -27,14 +27,55 @@
 """
 
 import sys
-from contextlib import contextmanager
-from io import BufferedReader, BytesIO, StringIO, TextIOBase, TextIOWrapper
+from contextlib import contextmanager, ExitStack, suppress
+from io import BufferedRandom, BufferedReader, BytesIO, TextIOBase, TextIOWrapper
 
 import attr
 
 
+class _SafeCloseIOBase(TextIOBase):
+    """Class to ignore ValueError during close.
+
+    Subclass of :cls:`~io.TextIOBase` that disregards ValueError which can
+    occur if the file has already been closed.
+    """
+
+    def close(self):
```

Comment:
Is this the correct way to do this? Or should this behavior be implemented as part of an `.__exit__()` override, especially since they're now [being used as context managers](https://github.com/bskinn/stdio-mgr/pull/26/files#diff-abadc7d46c647e14e338ebfbc175cc05R239-R241)?

(Or, maybe it doesn't matter?)

Reply:
It doesnt matter much, but __exit__ reduces our modifications of the streams behaviour.